guides/advanced_guides/keys.md

```diff
@@ -28,10 +28,9 @@ If no master key is given, all routes in MeiliSearch can be accessed without a k
 
 ### Reset keys
 
-Since the private and public keys are generated based on your master key, you must change the master key to reset the two other ones.
+Since the private and public keys are generated based on your master key, you must change the master key to change the two other ones.
 
 This means that you will have to restart MeiliSearch with the new master key.
 
-**All key will be changed**. This means that a single key can not be revoked.
+**All key will be changed**. This means that a single key can not be changed.
```

Comment:
2 times this means. If you can remove one it's cool, if not, we will see that later

Reply:
Changed !